ajo/personalization-client-side/README.md

```diff
@@ -0,0 +1,114 @@
+# Client-side Personalization
+
+## Overview
+
+This sample demonstrates using Adobe Experience Platform to get personalization content from Adobe Journey Optimizer. The web page changes based on the personalization content returned.
+
+This sample uses the [Adobe Experience Platform Web SDK](https://experienceleague.adobe.com/docs/experience-platform/edge/home.html) to get personalization content and to render it entirely client-side.
+
+Here is what the page looks like before and after personalization content is rendered.
+
+| without personalization                                        | with personalization                                                 |
+|----------------------------------------------------------------|----------------------------------------------------------------------|
+| <img src="../../.assets/plain.png" alt="drawing" width="800"/> | <img src="../../.assets/with-offers.png" alt="drawing" width="800"/> |
+
+Please review the [summary of personalization content](../AJOCampaigns.md) for this sample.
+
+Keep in mind that personalization content can come from any Adobe source such as [Journey Optimizer](https://experienceleague.adobe.com/docs/journey-optimizer/using/ajo-home.html?lang=en) or [Target](https://experienceleague.adobe.com/docs/target.html). Adobe Journey Optimizer is the source of personalization content used in this sample, but the implementation described here will work for all personalization solutions Adobe offers.
+
+## Running the sample
+
+<small>Prerequisite: [install node and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).</small>
+
+To run this sample:
+
+1. [Setup local SSL certificates for https](../../LocalSSLCertificateSetup.md).
+2. Clone the repository to your local machine.
+3. Open a terminal and change directory to this sample's folder.
+4. Run `npm install`
+5. Run `npm start`
+6. Open a web browser to [https://127.0.0.1](https://127.0.0.1)
```

Comment:
What is the reason you changed this to 127.0.0.1 instead of localhost?

Reply:
There are couple of inconsistencies here, 

Trailing Slash Problem in the surface (web://127.0.0.1/)
Personalization doesn’t work If page is loaded as (https://localhost/)

I've made a note on this to discuss further

Reply:
@krishnanbalasubramanyam are you saying you can't use `localhost` for AJO code-based campaigns?